.github/workflows/rust-lint.yml

```
@@ -66,4 +66,4 @@ jobs:
       - name: Run dependencies checks
         run: |
           gem install toml-rb --no-document
-          ruby scripts/linters/find_duplicate_deps.rb && ruby scripts/linters/find_unused_deps.rb
+          ruby scripts/linters/find_duplicate_deps.rb && ruby scripts/linters/find_unused_deps.rb && ruby scripts/linters/find_banned_deps.rb
```

let's make it a bit more readable.
```suggestion
          ruby scripts/linters/find_duplicate_deps.rb
          ruby scripts/linters/find_unused_deps.rb
          ruby scripts/linters/find_banned_deps.rb
```